pkg/types/spec/api.go

```diff
@@ -41,12 +41,14 @@ type API struct {
 
 	Key string `json:"key"`
 
+	CreatedTime  int64  `json:"created_time"`
```

Comment:
I would call this field InitialDeploymentTime or something to indicate that this is not just when this particular spec was created.